pkg/flowaggregator/flowaggregator.go

```diff
@@ -263,6 +372,15 @@ func (fa *flowAggregator) sendTemplateSet(templateSet ipfix.IPFIXSet) (int, erro
 	if err != nil {
 		return 0, fmt.Errorf("AddRecord failed, error: %v", err)
 	}
-	bytesSent, err := fa.exportingProcess.AddSetAndSendMsg(ipfixentities.Template, templateSet.GetSet())
+	bytesSent, err := fa.exportingProcess.SendSet(templateSet.GetSet())
 	return bytesSent, err
 }
+
+func (fa *flowAggregator) sendDataSet(dataSet ipfix.IPFIXSet) (int, error) {
+	sentBytes, err := fa.exportingProcess.SendSet(dataSet.GetSet())
+	if err != nil {
+		return 0, fmt.Errorf("error in IPFIX exporting process when sending data record: %v", err)
+	}
+	klog.V(4).Infof("Flow record created and sent. Bytes sent: %d", sentBytes)
+	return sentBytes, nil
+}
```

Comment:
I think the terminology should be consistent: are we sending a "flow record" or a "data set" (which I assumed was a collection of flow records)?

Reply:
We basically build up the flow record and send it within `SendSet`. Since there is currently only one set in the flow record, sending set and flow record refers to the same thing.